visualizer/src/Controls/ImageControls/RawControls/ColorPalette.js

```diff
@@ -0,0 +1,64 @@
+/** Modified from https://github.com/hms-dbmi/viv */
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import LensIcon from '@material-ui/icons/Lens';
+import React from 'react';
+
+// TODO: move to constants
+const COLOR_PALLETE = [
+  '#0000FF',
+  '#00FF00',
+  '#FF00FF',
+  '#FFFF00',
+  '#FF8000',
+  '#00FFFF',
+  '#FFFFFF',
+  '#FF0000',
+];
+
+const useStyles = makeStyles(() => ({
+  container: {
+    width: '70px',
+    height: '40px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+  },
+  button: {
+    padding: '3px',
+    width: '16px',
+    height: '16px',
+  },
+  icon: {
+    width: '17px',
+    height: '17px',
+    // border: '1px solid black',
+    // borderRadius: '50%'
+  },
```

Comment:
If possible it would be nice to make this related to the theme instead of using the pixel sizes directly.

I think `em` units are better for responsive web pages as well.

Reply:
This has been changed for our own components like the `SelectedPalette`. This component `ColorPalette` was adapted from another project https://github.com/hms-dbmi/viv, so I'll leave the specific pixel values for now